added SUSE and openSUSE (#260)
ossf/osv-schema: docs/schema.md

        </ul>
      </td>
    </tr>
    <tr>

>       <td><code>SUSE and openSUSE</code></td>

Reviewer: This needs to be the record prefix, which from a brief eyeball of https://lists.suse.com/pipermail/sle-security-updates/ should be `SUSE-SU`?

```suggestion
      <td><code>SUSE-SU</code></td>
```

Author: changed, with and not in code

      <td><a href="https://www.suse.com/support/security/">SUSE Security Landing page</a></td>
      <td>
        <ul>
          <li>How to contribute: <a href="https://www.suse.com/support/security/contact/">https://www.suse.com/support/security/contact/</a></li>
          <li>Source URL: <code>https://www.suse.com/support/update/</code></li>
          <li>OSV Formatted URL: <code>https://ftp.suse.com/pub/projects/security/osv/</code></li>
        </ul>
      </td>

>     </tr>

Reviewer 2: Would you be able to add some ecosystem definitions here: https://github.com/ossf/osv-schema/blob/516347895e3541e04db8166f9c83a22250f5b44d/docs/schema.md#affectedpackage-field also ?

There's a few other Linux distros in there that could potentially be useful to base off.

Author: I added stuff there

| `npm` | The NPM ecosystem; the `name` field is an NPM package name.  |
| `NuGet` | The NuGet package ecosystem. The `name` field is a NuGet package name.  |
| `OSS-Fuzz` | For reports from the OSS-Fuzz project that have no more appropriate ecosystem; the `name` field is the name assigned by the OSS-Fuzz project, as recorded in the submitted fuzzing configuration.  |

> | `openSUSE` | The openSUSE ecosystem; The ecosystem specifier is the marketing name of the openSUSE distribution. With the `:` removed, it matches the value in the /etc/os-release `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is a `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific openSUSE distribution. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|

Reviewer: Minor grammatical and styling:

```suggestion
| `openSUSE` | The openSUSE ecosystem; The ecosystem specifier is the marketing name of the openSUSE distribution. With the `:` removed, it matches the value in the `/etc/os-release` `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is an `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific openSUSE distribution. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|
```

> | `openSUSE` | The openSUSE ecosystem; The ecosystem specifier is the marketing name of the openSUSE distribution. With the `:` removed, it matches the value in the /etc/os-release `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is a `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific openSUSE distribution. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|

Reviewer 2: ```suggestion
| `openSUSE` | The openSUSE ecosystem; The ecosystem string has a `:<RELEASE>` suffix representing the marketing name of the openSUSE distribution. `<RELEASE>` matches the value in the /etc/os-release `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is a `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific openSUSE distribution. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|
```

Reviewer 2: (One more suggestion for consistency with other ecosystem definitions)

| `PyPI` | the Python PyPI ecosystem; the `name` field is a [normalized](https://www.python.org/dev/peps/pep-0503/#normalized-names) PyPI package name.  |
| `Red Hat` | The Red Hat package ecosystem; the `name` field is the name of a binary or source RPM. The ecosystem string has a `:<CPE>` suffix to scope the RPM to a specific Red Hat product stream. `<CPE>` is a translation of a Red Hat [Common Platform Enumerations](https://cpe.mitre.org/) (CPE) with the `cpe/:[oa]:(redhat):` prefix removed (for example, `Red Hat:rhel_aus:8.4::appstream` translates to `cpe:/a:redhat:rhel_aus:8.4::appstream`). Red Hat ecosystem identifiers can be used to identify vulnerable RPMs installed on a Red Hat system as explained [here](https://www.redhat.com/en/blog/how-accurately-match-oval-security-data-installed-rpms).  |
| `Rocky Linux` | The Rocky Linux package ecosystem; the `name` is the name of the source package. The ecosystem string might optionally have a `:<RELEASE>` suffix to scope the package to a particular Rocky Linux release. `<RELEASE>` is a numeric version.
| `RubyGems` | The RubyGems ecosystem; the `name` field is a gem name.  |

> | `SUSE` | The SUSE ecosystem; The ecosystem specifier is the marketing name of the SUSE product. With the `:` removed, it matches the value in the /etc/os-release `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is a `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific SUSE product. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|

Reviewer 2: ```suggestion
| `SUSE` | The SUSE ecosystem; The ecosystem string has a `:<RELEASE>` suffix representing the marketing name of the SUSE product. `<RELEASE>` matches the value in the /etc/os-release `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is a `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific SUSE product. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|
```

| `npm` | The NPM ecosystem; the `name` field is an NPM package name.  |
| `NuGet` | The NuGet package ecosystem. The `name` field is a NuGet package name.  |
| `OSS-Fuzz` | For reports from the OSS-Fuzz project that have no more appropriate ecosystem; the `name` field is the name assigned by the OSS-Fuzz project, as recorded in the submitted fuzzing configuration.  |

> | `openSUSE` | The openSUSE ecosystem; The ecosystem specifier is the marketing name of the openSUSE distribution. With the `:` removed, it matches the value in the `/etc/os-release` `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is an `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific openSUSE distribution. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|

Reviewer 2: Would you be able to make this consistent with the text for `SUSE` ? i.e. "The ecosystem string has a `:<RELEASE>` suffix representing the marketing name of the openSUSE distribution..."

Author: done so
